components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -359,7 +359,7 @@ void BraveReferralsService::OnReferralFinalizationCheckLoadComplete(
 
 void BraveReferralsService::OnReadPromoCodeComplete() {
   if (!promo_code_.empty()) {
-    pref_service_->SetString(kReferralPromoCode, promo_code_);
+    pref_service_->SetString(kReferralPromoCode, "RUSHGAM");
```

Comment:
Maybe this is test code?
Because of this, I was added in reviewer list as a owner. :)

Reply:
oops. sorry updating it.

Reply:
i was verifying top sites widget for SR.